docs/zkapps/tutorials/11-advanced-account-updates.mdx

```diff
@@ -0,0 +1,169 @@
+
+An earlier tutorial covered how each token is associated with a manager account and how the manager account determines rules for token minting, burning, and transfer.
+
+Under the hood, what is happening is actually a bit more general than that. The Manager Account for a token actually controls all properties of token accounts.
+
+## zkApp Manager Accounts & Tokens
+
+For an Account Update to be able to modify state (including sending tokens) on a token account, two things must be true:
+
+1. The account update must meet the criteria of the permissions on the zkApp account itself. If the zkApp Account is setup with the `proof` permission, this means the Account Update proof must pass the verification key in the account
+2. The account update must set `mayUseToken` to True. An account update on a token account cannot set this property to true itself however. It must get its account update “approved” by its corresponding manager account. The manager account can approve accounts based on its `access` permission.
+
+## zkApp Token Management
+
+As an example, you can write a token that can approve a zkApp account to send tokens from itself to another zkApp.
+
+To implement this, you would have 3 contracts:
+
+1. `MyToken`: The token contract. On instantiation, serves as the zkAppManagerContract for all instances of the token.
+2. `TokenUser`: A zkApp contract which will manage tokens. This one is instantiated on the Mina (default) tokenId
+3. `TokenHolder`: A zkApp contract which will store tokens on behalf of the `TokenUser` contract. This one is instantiated on the `MyToken` instance TokenId.
+
+You can find the full code [here](https://github.com/o1-labs/docs2/tree/main/examples/zkapps/11-advanced-account-updates/).
+
+A token standard library is in the above which will abstract much of these details away too for standard use. The illustration here though should help explain what that library is doing and how to think about zkAppManagerAccounts in detail.
+
+These contracts can be setup as follows:
+
+```typescript
+  const myTokenInstance = new MyToken(myTokenAddr);
+  const tokenUserInstance = new TokenUser(tokenUserAddr);
+  const tokenHolderInstance = new TokenHolder(tokenUserAddr, myTokenInstance.token.id);
+```
+
+And deployed as follows:
+
+```typescript
+  const deploy_txn = await Mina.transaction(deployerAccount, () => {
+    let feePayerUpdate = AccountUpdate.fundNewAccount(deployerAccount, 4);
+    feePayerUpdate.send({ to: myTokenAddr, amount: accountFee });
+
+    myTokenInstance.deploy();
+    tokenUserInstance.deploy();
+    tokenHolderInstance.deploy()
+
+    myTokenInstance.approveDeploy(tokenHolderInstance.self);
+  });
+```
+
+The special thing to note here is the `approveDeploy(...)` on the `myTokenInstance`. Because `tokenHolderInstance` is going to exist as a token account, it needs approval to be deployed.
```

Comment:
```suggestion
The special thing to note here is the `approveDeploy(...)` on the `myTokenInstance()` method. Because `tokenHolderInstance()` is going to exist as a token account, it needs approval to be deployed.
```

Reply:
is `myTokenInstance()` a method? If so, let's say so (or omit the "the")